stdlib/public/Concurrency/Clock.swift

```diff
@@ -40,51 +40,61 @@ public protocol Clock<Duration>: Sendable {
 
 #if !SWIFT_STDLIB_TASK_TO_THREAD_MODEL_CONCURRENCY
   func sleep(until deadline: Instant, tolerance: Instant.Duration?) async throws
-#endif
-
-  /// The traits associated with this clock instance.
-  @available(StdlibDeploymentTarget 6.2, *)
-  var traits: ClockTraits { get }
 
-  /// Convert a Clock-specific Duration to a Swift Duration
-  ///
-  /// Some clocks may define `C.Duration` to be something other than a
-  /// `Swift.Duration`, but that makes it tricky to convert timestamps
-  /// between clocks, which is something we want to be able to support.
-  /// This method will convert whatever `C.Duration` is to a `Swift.Duration`.
+  /// Run the given job on an unspecified executor at some point
+  /// after the given instant.
   ///
   /// Parameters:
   ///
-  /// - from duration: The `Duration` to convert
+  /// - job:         The job we wish to run
+  /// - at instant:  The time at which we would like it to run.
```

Comment:
does spelling params like this actually parse well in docc? I thought one has to put only the second name 🤔